### ISSUE TITLE
suppress iris warnings

### ISSUE THREAD

**Issue by DamienIrving** [DamienIrving/ocean-analysis#13]
```
with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    cube = iris.load_cube('infile.nc')
```
